SSLeay.xs

```diff
@@ -1474,6 +1474,71 @@ void ssleay_ctx_info_cb_invoke(const SSL *ssl, int where, int ret)
     LEAVE;
 }
 
+void ssleay_msg_cb_invoke(int write_p, int version, int content_type, const void *buf, size_t len, SSL *ssl, void *arg)
+{
+    dSP;
+    SV *cb_func, *cb_data;
+
+    cb_func = cb_data_advanced_get((void*)ssl, "ssleay_msg_cb!!func");
```

Comment:
I think these two casts to `void *` are not needed in this case. The info callback seems to need it because SSL there is a pointer to const.

Reply:
They are indeed not needed there, I'll remove those casts.